src/Main/OrganizationsFilter.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import { AccordionSet } from '@folio/stripes/components';
+import { AcqCheckboxFilter } from '@folio/stripes-acq-components';
+
+import COUNTRY_OPTIONS from '../Utils/Country';
+import LANGUAGE_OPTIONS from '../Utils/Languages';
+
+import {
+  FILTERS,
+  BOOLEAN_OPTIONS,
+  CATEGORY_OPTIONS,
+  STATUS_OPTIONS,
+  PAYMENT_METHOD_OPTIONS,
+} from './constants';
+
+class OrganizationsFilter extends Component {
```

Comment:
should it be a `Component`? IMHO, SFC works fine here

Reply:
100%